account_move_line_search_extension/account.py

```diff
@@ -47,7 +47,8 @@ def search(self, cr, uid, args, offset=0, limit=None, order=None,
         if context and 'account_move_line_search_extension' in context:
             ana_obj = self.pool['account.analytic.account']
             for arg in args:
-                if arg[0] == 'analytic_account_id':
+                if arg[0] == 'analytic_account_id' and\
+                   isinstance(arg[0], basestring):
```

Comment:
And the strongest reason is pointed in a comment: "Note that the trailing \ solutions are not recommended by PEP 8. One reason is that if a space is added by mistake after a \ it might not show in your editor, and the code becomes syntactically incorrect."